types/react-native/index.d.ts

```diff
@@ -162,6 +162,31 @@ interface EmitterSubscription extends EventSubscription {
     remove(): void;
 }
 
+interface DEPRECATED_RCTExport<T> {
+	getConstants(): Function;
+}
+
+interface TurboModule extends DEPRECATED_RCTExport<void> {}
+
+interface Spec extends TurboModule {
+    addListener(eventName: string): void;
+    removeListeners(count: number): void;
+}
+
+declare class TurboModuleRegistry {
+	get<TurboModule>(name: string): (legacyModule: any) => TurboModule
```

Comment:
[The upstream Flow type](https://github.com/facebook/react-native/blob/v0.61.5/Libraries/TurboModule/TurboModuleRegistry.js#L19) states that the return type should _extend_ the `TurboModule` interface, in your case you simply _named_ the type parameter `TurboModule`. Also, I’m not sure where your returned function `(legacyModule: any) => TurboModule` comes from, I don’t see that in the source.

```suggestion
	get<T extends TurboModule>(name: string): T | null | undefined
```